PharoTour/PharoTour.pillar

```diff
@@ -368,12 +367,22 @@ When a new System Browser window first opens, all panes are empty but the
 leftmost one. This first pane lists all known packages, which contain groups of
 related classes.
 
+@@todo TODO: BTC-2015-08-26 double-check this next bit
```

Comment:
please use @@authorToDo to indicate something the author should do. Only use @@todo when the reader should do something